service_mesh/service_mesh_install/installing-ossm.adoc

```diff
@@ -4,30 +4,31 @@ include::modules/ossm-document-attributes.adoc[]
 :context: installing-ossm
 toc::[]
 
-Installing the {ProductShortName} involves installing the Elasticsearch, Jaeger, Kiali and {ProductShortName} Operators, creating and managing a `ServiceMeshControlPlane` resource to deploy the control plane, and creating a `ServiceMeshMemberRoll` resource to specify the namespaces associated with the {ProductShortName}.
+Installing the {ProductShortName} involves using the link:https://operatorhub.io/[OperatorHub] to install the Elasticsearch, Jaeger, Kiali, and {ProductShortName} Operators.
```

Comment:
If using the auto install of the other operators, we could just simplify this type of statement so it doesn't refer to the other operators? So from the user's perspective they are just installing the Service Mesh operator?